tsconfigdoc.json

```diff
@@ -0,0 +1,254 @@
+{
+  "$schema": "http://json.schemastore.org/tsconfig",
+  "extends": "./tsconfigbase",
+  "compilerOptions": {
+    "composite": false,
+    "baseUrl": ".",
+    "paths": { "@jupyterlab/*": ["packages/*/src"] }
+  },
+  "references": [
```

Comment:
We should generate this in `buildutils/src/ensure-repo.ts`

Reply:
I agree

Reply:
autogen enabled